src/index.tsx

```diff
@@ -8,7 +8,7 @@ export interface UseSuperclusterArgument<P, C> {
   points: Array<Supercluster.PointFeature<P>>;
   bounds?: BBox;
   zoom: number;
-  options?: Supercluster.Options<P, C>;
+  options?: Supercluster.Options<P, C> & { enabled?: boolean };
```

Comment:
Do you think we could separate this out from `Supercluster.Options`?

What exactly are we disabling, we are disabling supercluster updating which points are loaded and/or the clusters given the bounds/zoom level.

Could we name it something like `disableRefresh` and you set it to true if you want to stick with the prior version?

Reply:
While it's true that Supercluster.Options supporting the 'disableRefresh' option would be ideal, I personally think there isn't sufficient justification for Supercluster to support 'disableRefresh.'

In vanilla JavaScript, synchronous processing, as shown in the pseudo code below, is possible, making 'disableRefresh' unnecessary.

```javascript
 // vanilla javascript
googleMap.addListener('zoomout', async (newMapInformation) => { 
  const res = await fetch('/getMarkers', {
    newMapInformation
  });

  const clusters = supercluster.getClusters(newMapInformation);
  updateView(clusters);
})
```

In the case of React, as evident in the following code, it appears that 'disableRefresh' is necessary due to re-rendering situations.

```JSX
const MapComponent = () => {
  const [mapInformation, setMapInformation] = useState(initialMapInformation);

  const { data } = useQuery("markers", getMarkers);

  const { clusters } = useSupercluster({
    ...newMapInformation
  });

  // react
  return (
    <GoogleMap
      onZoomout={newMapInformation => {
        setMapInformation(newMapInformation);
      }}
    >
      {clusters.map(() => {
        return <Marker />; // draw clustered markers
      })}
    </GoogleMap>
  );
};
```

I will reflect the variable name change